packages/data-models/constants.ts

```diff
@@ -101,6 +106,7 @@ const APP_HEADER_DEFAULTS = {
 };
 
 const APP_SIDEMENU_DEFAULTS = {
+  enable_sidemenu: true,
```

Comment:
thanks for adding this, although as it's already the side_menu defaults probably just calling it `enabled` would be sufficient